face.go

```diff
@@ -106,7 +118,14 @@ func (rec *Recognizer) recognize(imgData []byte, maxFaces int) (faces []Face, er
 	return
 }
 
-func (rec *Recognizer) recognizeFile(imgPath string, maxFaces int) (face []Face, err error) {
+func Squared_euclidean_distance(sample Descriptor, testSample Descriptor) (distance float32) {
```

Comment:
We can implement it in pure Go, see https://en.wikipedia.org/wiki/Euclidean_distance#Squared_Euclidean_distance

Reply:
Also should use CamelCase for naming